OpenRA.Game/Server/Server.cs

```diff
@@ -29,8 +29,9 @@ namespace OpenRA.Server
 	public enum ServerState
 	{
 		WaitingPlayers = 1,
-		GameStarted = 2,
-		ShuttingDown = 3
+		GameLoading = 2,
```

Comment:
Can we please track this using a bool inside the `GameStarted` state instead?

This enum is part of the master server API, so changing these values would require a new protocol version and changes to the masterserver and online server list. A big task for a small feature.

In the future I want to allow clients to share their current loading percentage, so that bool would be replaced by a dictionary of client to number.

Reply:
Sure thing